Tasks/google-play-promote/task.json

```diff
@@ -109,6 +109,14 @@
             "required": false,
             "helpMarkDown": "The percentage of users the specified APK will be released to for the specified 'Destination track'. It can be increased later with the 'Google Play - Increase Rollout' task.",
             "visibleRule": "rolloutToUserFraction = true"
+        },
+        {
+            "name": "сleanTheSourceTrack",
+            "type": "boolean",
+            "label": "Сlean the source track",
+            "defaultValue": true,
```

Comment:
If this is a new behavior, we should default it to false so customers have to opt-in to get it. Not sure if it would break existing users if it was defaulted to on for them when they upgraded the extension

Reply:
Nevermind, disregard this comment